docs/shared/packages/next/next-config-setup.md

```diff
@@ -125,7 +125,8 @@ module.exports = async (phase, context) => {
 
   // If you have plugins that has to be added after Nx you can do that here.
   // For example, Sentry needs to be added last.
-  updatedConfig = require('@sentry/nextjs')(updatedConfig, { silent: true });
+  const { withSentryConfig } = require('@sentry/nextjs');
+  updatedConfig = withSentryConfig(updatedConfig, { silent: true });
```

Comment:
I would also suggest turning `silent` off because it mutes useful debugging information but I will leave that up to you.

Reply:
Actually, I went ahead and removed the silent because the log output is actually too useful to be silenced by default.